lib/faktory_worker/worker.ex

```diff
@@ -9,6 +9,7 @@ defmodule FaktoryWorker.Worker do
   @type t :: %__MODULE__{}
 
   @fifteen_seconds 15_000
+  @faktory_default_reserve_for 1800
```

Comment:
Do we need this if we're setting the default reserve_for on the Job module?

Reply:
This is covering an edge case I came up with.

Originally I was taking this value from the job module. However there is a case where the job was sent to faktory before having a custom `reserve_for` set on the module meaning we would potentially use the incorrect value that is set on faktory.